src/DotLiquid.Tests/CustomIndexableTests.cs

```diff
@@ -87,7 +87,7 @@ public void TestCustomIndexableIntKeys()
         {
             string output = Template.Parse("1: {{container[0]}}, 2: {{container[1]}}")
                 .Render(Hash.FromAnonymousObject(new {container = new CustomIndexable()}));
-            Assert.AreEqual("1: System.Int32 0, 2: System.Int32 1", output);
+            Assert.AreEqual("1: System.Int64 0, 2: System.Int64 1", output);
```

Comment:
Keep index keys within int range just 32 bit ints.

Reply:
Long to be used when the index exceeds 32 bit integer range, which is ok for arrays (.NET allows very large arrays which accessor index is long)